liquid_node/configuration.py

```diff
@@ -250,6 +259,8 @@ def __init__(self):
         self.matrix_session_lifetime = self.ini.get('matrix', 'session_lifetime', fallback='168h')
         self.matrix_media_lifetime = self.ini.get('matrix', 'media_lifetime', fallback='365d')
         self.matrix_message_lifetime = self.ini.get('matrix', 'message_lifetime', fallback='365d')
+        self.bbb_endpoint = self.ini.get('bbb', 'bbb_endpoint', fallback='')
+        self.bbb_secret = self.ini.get('bbb', 'bbb_secret', fallback='')
```

Comment:
add an assert here (or lower) that checks both these params are set to non-empty values if BBB is on. Assert string should state that `[bbb]bbb_endpoint, _secret must be set when bbb is turned on`


Since this requires manual configuration, the default ini file will crash at boot. We don't want that to happen. Make bbb be off by default, even if `default_app_status = on` -- check `grist` and `prophecies` on how it's done (a small change in the `is_enabled` fn)